src/types/ItemsPerPage.ts

```diff
@@ -0,0 +1,6 @@
+export enum ItemsPerPageType {
+  Three = 3,
+  Five = 5,
+  Ten = 10,
+  Twenty = 20,
+}
```

Comment:
It will be more universal to use names without reference to numbers:
```suggestion
export enum PageSize {
  Small = 3,
  Medium = 5,
  Large = 10,
  ExtraLarge = 20,
}
```